ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-security.j2

```diff
@@ -129,6 +129,29 @@
 | {{ policy.name }} | > {{ min_digits }} | > {{ min_length }} | > {{ min_lower }} | > {{ min_special }} | > {{ min_upper }} | < {{ max_repetitive }} | < {{ max_sequential }} |
 {%         endfor %}
 {%     endif %}
+{%     if management_security.shared_secret_profiles is arista.avd.defined %}
+
+### Secret Profiles
+{%         for profile in management_security.shared_secret_profiles | arista.avd.natural_sort("profile") %}
+
+#### {{ profile.profile }}
+
+| Secret Name | Receive Lifetime | Transmit Lifetime | Timezone |
+| ----------- | ---------------- | ----------------- | -------- |
+{%             for secret in profile.secrets | arista.avd.natural_sort("name") %}
+{%                 set local_time = "Local Time" if secret.local_time is arista.avd.defined(true) %}
+{%                 if secret.receive_lifetime.infinite is arista.avd.defined and secret.transmit_lifetime.infinite is arista.avd.defined %}
```

Comment:
Simplify the logic by using variables for `secret.receive_lifetime` and `secret.transmit_lifetime` and keeping the table only once

Reply:
Almost there .. :)

Reply:
Fixed!